robot-server/robot_server/runs/run_store.py

```diff
@@ -7,6 +7,7 @@
 from .action_models import RunAction, RunActionType
 
 from robot_server.persistence import run_table, action_table
+from robot_server.protocols.protocol_store import ProtocolNotFoundError
```

Comment:
Should I move ProtocolNotFound to a shared location?

Reply:
It's already a public part of the `robot_server.protocols` module's interface. Import it from there instead of doing a deep import from `protocol_store`

```suggestion
from robot_server.protocols import ProtocolNotFoundError
```